src/interface.ts

```diff
@@ -14,6 +14,14 @@ export interface IDiff {
 	is_deleted?: boolean;
 }
 
+export interface ITab {
+	repo_id: number;
+	created_at: string;
+	source: string;
+	file_name: string;
+	tabs: any[];
```

Comment:
We can define another interface for this as 

```
ITabFiile {
    file_id: number | null
    path: string
}
```

Reply:
Won't it be simpler to append tabs string object to `ITab.tabs`?

Reply:
Wouldn't be type scripted I think